content/influxdb/v2.0/backup-restore/restore.md

```diff
@@ -20,25 +20,103 @@ Use the `influx restore` command to restore backup data and metadata from Influx
+Use the `--new-bucket` flag to specify the new bucket to restore data to.
+To restore data and retain bucket names, [delete existing buckets](/influxdb/v2.0/organizations/buckets/delete-bucket/)
+and then begin the restore process.
+{{% /note %}}
+
 ## Restore data with the influx CLI
-Use the `influx restore` command and specify the path to the backup directory in the first argument.
+Use the `influx restore` command and specify the path to the backup directory.
+
+_For more information about restore options and flags, see the
+[`influx restore` documentation](/influxdb/v2.0/reference/cli/influx/restore/)._
+
+- [Restore all time series data](#restore-all-time-series-data)
+- [Restore data from a specific bucket](#restore-data-from-a-specific-bucket)
+- [Restore and replace all InfluxDB data](#restore-and-replace-all-influxdb-data)
+
+### Restore all time series data
+To restore all time series data from a bucket directory into buckets with the
+same name as the backup source buckets, provide the following:
+
+- backup directory path
+
+```sh
+influx restore \
+  --input /backups/2020-01-20_12-00/
+```
+
+### Restore data from a specific bucket
+To restore data from a specific bucket in a backup directory, provide the following:
+
+- backup directory path
+- bucket name or ID
+
+```sh
+influx restore \
+  --input /backups/2020-01-20_12-00/ \
+  --bucket example-bucket
+
+# OR
+
+influx restore \
+  --input /backups/2020-01-20_12-00/ \
+  --bucket-id 000000000000
+```
+
+If a bucket with the same name as the backed up bucket already exists, use the
+`--new-bucket` flag to specify a new bucket to restore the data to.
 
 ```sh
-# Syntax
-influx restore <path-to-backup-directory>
+influx restore \
+  --input /backups/2020-01-20_12-00/ \
+  --bucket example-bucket \
+  --new-bucket new-example-bucket
+```
+
+### Restore and replace all InfluxDB data
+To restore and replace all time series data _and_ InfluxDB key-value data such as
+tokens, users, dashboards, etc., include the following:
 
-# Example
-influx restore ~/backups/2020-01-20_12-00/
+- `--full` flag
+- backup directory path
+
+```sh
+influx restore \
+  --input /backups/2020-01-20_12-00/ \
+  --full
 ```
 
-_For more information about restore options and flags, see the
-[`influx restore` documentation](/influxdb/v2.0/reference/cli/influx/restore/)._
+{{% note %}}
+#### Restore to a brand new InfluxDB server
+If using a backup to populate a new InfluxDB server:
+
+1. Retrieve the [admin token](/influxdb/v2.0/security/tokens/#admin-token) from your source InfluxDB instance.
+2. Set up your new InfluxDB instance, but use the `-t`, `--token` flag to use the
+   **admin token** from your source instance as the admin token on your new instance.
+
+    ```sh
+    influx setup --token My5uP3rSecR37t0keN
+    ```
+3. Restore the backup to the new server.
+
+    ```sh
+    influx restore \
+      --input /backups/2020-01-20_12-00/ \
+      --full
+    ```
```

Comment:
Should we add a "if you don't do this, the restore process will fail in XYZ way..." section below this explaining why?

Reply:
Sure, I can add that.